src/NServiceBus.RavenDB.Tests/Timeouts/When_adding_timeouts.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace NServiceBus.RavenDB.Tests.Timeouts
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus.Extensibility;
+    using NServiceBus.TimeoutPersisters.RavenDB;
+    using NUnit.Framework;
+    using TimeoutData = NServiceBus.Timeout.Core.TimeoutData;
+
+    public class When_adding_timeouts : RavenDBPersistenceTestBase
+    {
+        [Test]
+        public async Task Add_WhenNoIdProvided_ShouldSetDbGeneratedTimeoutId()
```

Comment:
good question. It's required in the unit tests but that shouldn't be an argument. But being able to add something to a persistence store and the only way to get that out is by using a query from a different interface looks like a shitty API to me. But I guess we have several options here. We did talk about removing auto-id-generation anyway, right?